master/pkg/searcher/search_method.go

```diff
@@ -34,11 +35,19 @@ type SearchMethod interface {
 	trialExitedEarly(
 		ctx context, requestID model.RequestID, exitedReason model.ExitedReason,
 	) ([]Operation, error)
+
 	// TODO: refactor as model.Snapshotter interface or something
 	model.Snapshotter
 	expconf.InUnits
 }
 
+// CustomSearchMethod is the interface for the custom search method.
+type CustomSearchMethod interface {
```

Comment:
Hang on, does this actually need to be an interface? I know I previously suggested switching this to a separate interface when it was previously just part of the `SearchMethod` interface, but do we actually need it at all?